### ISSUE TITLE
Automating regular website checks

### ISSUE THREAD

**Issue by KaKi87** [kevquirk/512kb.club#1187]
If I understand the FAQ correctly, regular website checks are done manually, but did you consider automating these ?

**Comment by kevquirk** [comment]
Sorry for the delay in reply - only just saw this.

We kinda do. [We have scripts](https://github.com/kevquirk/512kb.club/tree/main/scripts) we run to check them, but it's still fairly manual. If you have any ideas to fully automate it, I'd love to hear them. Unfortunately, a lot of the services out there that can do this kind of thing cost money.

**Comment by kevquirk** [comment]
No, we're not at the moment, because it's so manual. If we wanted to fully automate checking for both size changes and whether they're still up, we'd need to (I think).

**Comment by KaKi87** [comment]
A headless browser can do these as well.

**Comment by garritfra** [comment]
I think this thread is relevant: #749

I'll dig into that later today, it's been a while.

**Comment by NeonGamerBot-QK** [comment]
> A headless browser can do these as well.

this is def possible and can be done easily in a gh action.

**Comment by Raymi306** [comment]
https://github.com/kevquirk/512kb.club/pull/1390 should close this I think

**Comment by KaKi87** [comment]
Well, the script has yet to be automatically ran on a schedule, e.g. using GitHub Actions (in which case the authentication parameter must be moved into an environment variable).

**Comment by Raymi306** [comment]
Oh!
I did not realize that.
I may be able to assist writing an action, or with providing a server to run it from that has some spare bandwidth.

**Comment by kevquirk** [comment]
@moebrowne just submitted #1830 which solves a lot of this. I just need to find the time to look at their [512-check repo](https://github.com/moebrowne/512-check) and start using it myself.
